website/app/styles/doc-components/scroll-to-top.scss

```
@@ -0,0 +1,29 @@
+@use "../typography/mixins" as *;
+
+.doc-scroll-to-top {
+  position: fixed;
+  right: 40px;
+  bottom: 40px;
+  z-index: 99;
+  width: 44px;
+  height: 44px;
+  padding: 8px 8px 4px 8px;
+  background-color: rgba(52, 53, 54, 80%);
+  border: 2px solid transparent;
+  border-radius: 50%;
+  box-shadow: 0 8px 12px 0 rgba(37, 38, 45, 8%);
+  cursor: pointer;
+  transition: bottom 150ms linear;
+
+  &:hover {
+    bottom: 43px;
+    background-color: rgba(29, 30, 31, 80%);
+    box-shadow: 0 16px 28px 0 rgba(37, 38, 45, 12%);
+  }
+
+  &:focus {
+    border-color: white;
+    outline: none;
+    box-shadow: 0 0 0 3px var(--doc-focus-ring-color);
```

we tend not to use HTML names for colors, and in the Figma designs the shadow is still visible in the focused state (correct me if I'm wrong @heatherlarsen)
```suggestion
    border-color: #fff;
    outline: 3px solid var(--doc-focus-ring-color);
```